src/content/components/uikit/Switch/index.ts

```diff
@@ -0,0 +1,37 @@
+import dynamic from 'next/dynamic';
+
+import {Repos} from '../../../../types/common';
+import {getGithubUrl, getReadmeUrl, mappingOptions} from '../../utils';
+
+const getterOptions = {repoName: Repos.Uikit, componentName: 'Switch'};
+
+export const switchConfig = {
+    id: 'switch',
+    title: 'Switch',
+    githubUrl: getGithubUrl(getterOptions),
+    content: {
+        readmeUrl: getReadmeUrl(getterOptions),
+    },
+    sandbox: {
+        component: dynamic(() => import('./SwitchComponent').then((mod) => mod.SwitchComponent)),
```

Comment:
This should be replaced with:
`dynamic(() => import('@gravity-ui/uikit').then((mod) => mod.Switch))`

`SwitchComponent` doesn't have predefined props for `Switch` and can be deleted.

And the same situation with `CheckboxComponent`.